app/views/admin/api/_api_key_form.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= form_for @api_key,
+             url: @api_key.new_record? ? new_admin_api_path(@api_key) : admin_api_path(@api_key),
+             html: { class: "post" } do |f| %>
+  <%= error_messages_for @api_key %>
+
+  <p class="required notice">* <%= ts("Required information") %></p>
+
+  <fieldset>
+    <legend><%= ts("API Token") %></legend>
+    <h3 class="landmark heading"><%= ts("API Token") %></h3>
+    <dl>
+      <dt class="required"><%= f.label :name, ts("Name") + "*" %></dt>
+      <dd class="required">
+        <%= f.text_field :name %>
+        <%= live_validation_for_field("api_key_name",
+                                      :presence => true,
+                                      :maximum_length => Profile::PROFILE_TITLE_MAX) %>
+      </dd>
+      <dt><%= f.label :banned, ts("Banned?") %></dt>
+      <dd>
+        <%= f.check_box :banned %>
+      </dd>
```

Comment:
This should be the other way around -- the check box should be in the dt, then the label in the dd. (It's a terrible system either way, unfortunately, but consistently terrible at least has consistency going for it.)